Sources/ApodiniDatabase/Handlers/Create/Create.swift

```diff
@@ -13,4 +13,6 @@ public struct Create<Model: DatabaseModel>: Handler {
     public func handle() -> EventLoopFuture<Model> {
         object.save(on: database).transform(to: object)
     }
+    
+    public init() {}
```

Comment:
I think the other Handlers like `Delete` and `ReadOne` would also benefit from an initializer

Reply:
Good point! I've added the missing initializers